### ISSUE TITLE
[Svelte] Unnecessary required properties "id", "source", "target" in EdgeProps

### ISSUE THREAD

**Issue by futurGH** [xyflow/xyflow#3666]
### Describe the Bug

`EdgeProps`, and by extension the edge components, require the props `id`, `source`, and `target`, which aren't actually used anywhere within the components. This isn't the case for the equivalent types & components in react-flow, so it adds just a bit of unnecessary cruft when porting a project over to Svelte.

### Steps to Reproduce the Bug or Issue

```svelte
<script lang="ts">
import { BezierEdge } from "@xyflow/svelte";
</script>

<!-- missing properties: id, source, target -->
<BezierEdge />
```

### Expected behavior

Those props shouldn't be required given that `source` and `target` aren't used within any of the edge components as far as I can tell, and `id` is only passed through to `BaseEdge`, for which it's an optional property.

### Platform

- Svelte 4.2.7
- @xyflow/svelte 0.0.28

**Comment by peterkogo** [comment]
Good catch! This is indeed not necessary and due to the way we use BezierEdge internally. Will be fixed in the next release.

**Comment by moklick** [comment]
fixed in 0.0.29
